frontend/src/pages/teacherEditor/index.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import TeacherEditor from "../../components/teacherEditor";
+
+export default function TeacherEditorPage() {
+    return (
+        <React.Fragment>
+            <TeacherEditor/>
+        </React.Fragment>
```

Comment:
can use the `<></>` shorthand for react fragments

Reply:
Good to know